students/douglas_klos/session2/fizzbuzz.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+# Douglas Klos
+# January 17th, 2019
+# Python 210, Assignment 2
+# fizzbuzz.py
+
+# For loop is not inclusive, so we run it to 101 to include 100.
+def fizzbuzz(n=100):
+    """ displays fizzbuzz to specified number, defaults to 100 """
+    
+    # For loop is not inclusive, so we run it to 101 to include 100.
+    for num in range(1,n+1):
```

Comment:
you'll want to include a space after your commas.  Python has a particular "style" that acceptable to the general community which can be found here: https://www.python.org/dev/peps/pep-0008/

Reply:
Thank you for bringing the pep8 styling to my attention, I believe the code now conforms to those standards.  From what I read, it seems I wouldn't want white space around the * operator like below, from grid_printer.py, though it looks a bit strange to me like this.  (I don't know how to link directly into the file like you did yet...)  Would this be correct?

`    str1 = '+ ' + '- '*4 + '+ ' + '- '*4 + '+ ' + '\n'`
`    str2 = '| ' + '  '*4 + '| ' + '  '*4 + '| ' + '\n'`

`    print((('| ' + '  '*y) *x) + '|')`
`    print((('+ ' + '- '*y) *x) + '+\n')`

Thanks again

Reply:
You should also have a space between every operator `x + y * 2 / 3`.
```str1 = '+ ' + '- ' * 4 + '+ ' + '- ' * 4 + '+ ' + '\n'```